k8s/wordpress/chart/wordpress/templates/wordpress-manifests.yaml.template

```diff
@@ -13,6 +13,28 @@ stringData:
 data:
   wp-password: "{{ .Values.admin.password }}"
 ---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-wordpress-config
+  labels:
+    app.kubernetes.io/name: {{ .Release.Name }}
+    app.kubernetes.io/component: wordpress-webserver
+data:
+  .htaccess: |
+    # BEGIN WordPress
+    <IfModule mod_rewrite.c>
```

Comment:
Please add a short comment about the intention of this configuration.

Reply:
Done :+1: